Assets/Tests/PlayMode/JelliesParametersTests.cs

```diff
@@ -65,4 +65,17 @@ public IEnumerator TestIfJellyGetsHungrierAfter10Seconds()
         Assert.LessOrEqual(parameters.Satiation, parameters.MaxSatiation);
     }
     */
+
+    [UnityTest]
+    public IEnumerator TestMoodChangesAfterPet()
+    {
+        CreateTestJelly(out GameObject testJelly, out Parameters parameters);
+        parameters.ReqAmountOfPetsPerState = new int[1];
+        parameters.ReqAmountOfPetsPerState[0] = 1;
+
+        parameters.IncreasePetCounter();
+
+        yield return null;
+        Assert.AreEqual(MoodStates.Slightly_Needy, parameters.MoodState);
+    }
```

Comment:
We haven't been testing with unit tests when we review pull requests. Maybe we should discuss doing unit tests during a meeting.